### ISSUE TITLE
Killing the game via the console does not work if a wrapper script is used

### ISSUE THREAD

**Issue by QuImUfu** [MultiMC/MultiMC5#1747]
System Information:
-----------------------------
MultiMC5 0.4.12-dev-845 on Linux Lubuntu 64bit.

Summary of the issue or suggestion:
----------------------------------------------
Killing the game via the console window does not work if a wrapper script is used.
It kills the wrapper-progress instead.

What should happen:
------------------------------
There should be some way for MultiMC to get the child progresses of the wrapper command and kill them, too. 
I don't know whether there is a solution that works for all supported OSs.

**Comment by MggMuggins** [comment]
You probably don't want to kill _all_ of the child processes for the script. That could potentially cause some issues. I would say if the script doesn't exit if MC is killed, then that's the user's problem and they need to fix their script. The question here is how to get ahold of the PID (right?) of the java process that is running MC.

**Comment by QuImUfu** [comment]
Exactly, but as i think it is very hard to find out witch progress is the Minecraft one, killing the progress started by MultiMC and all child progresses would be the simplest solution that would for sure kill Minecraft as well. But i have to admit that it could lead to issues as well, but not more than the current behaviour (killing only the wrapper command).

**Comment by MggMuggins** [comment]
Would it be possible to specify some kind of specification for wrappers that would spit out a PID?

**Comment by QuImUfu** [comment]
Yeah, that would be possible, but what about wrapper commands you do not have (direct) access to (eg. for recording OpenGL applications)?
The best option would be getting the child progresses and killing the java and minecraft one. This would be possible under Linux, i don't know whether there is a solution for that that works cross-platform.

**Comment by MggMuggins** [comment]
Linux and OSX are going to be fairly similar in that regard, since OSX is _technically_ a unix. I really have no idea about Windows. Although I figure that out of all the users of MultiMC, most of those that use a startup script are running Linux...

**Comment by cocona20xx** [comment]
Can confirm that this still exists as of Late Oct. 2021 on macOS; if the game is already running and a wrapper is being used, killing the game from within multimc does not actually close it (you have to close it via quit/force-quit/killing the process in the Activity Monitor or via a terminal command)
